### ISSUE TITLE
`amplify env get --json` returns invalid json

### ISSUE THREAD

**Issue by matthiasotto** [aws-amplify/amplify-cli#1616]
**Describe the bug:**
I want to be able to easily parse details from my environment using common CLI tools like `jq` and `fx`. However, the output from `amplify env get` appears to deliver invalid json.

**To Reproduce:**
### jq
```
$ amplify env get --name staging --json | jq
parse error: Invalid numeric literal at line 1, column 2
```

### fx
```
$ amplify env get --name staging --json | fx
undefined:1
{
^

SyntaxError: Unexpected token n JSON at position 0
```

**Expected behavior:**
The `--json` option should be returning valid json.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Windows Bash (Ubuntu)
 - Amplify CLI Version 1.7.0

**Comment by kaustavghosh06** [comment]
Could you see what does `amplify env get --name <env-name> --json` return?

**Comment by matthiasotto** [comment]
> Could you see what does `amplify env get --name <env-name> --json` return?

Yes @kaustavghosh06 , it does looks like JSON to the naked eye...
<img width="1105" alt="Untitled" src="https://user-images.githubusercontent.com/6186593/59124531-2fc4e300-891d-11e9-9186-06a2085a289f.png">

However, when I drop it into a file it looks like there is encoding issues...

```
amplify env get --name staging --json > envGetOutput.json
```
<img width="1473" alt="Screen Shot 2019-06-07 at 12 15 19 PM" src="https://user-images.githubusercontent.com/6186593/59124829-f476e400-891d-11e9-9ab6-d84782de5ba1.png">

**Comment by kaustavghosh06** [comment]
the fix for this issue has been merged as a part of this PR - #1622 
It will be a part of our next CLI release - 1.7.2.
